client/src/components/clan/ForumEntry.jsx

```diff
@@ -9,24 +11,57 @@ import { Switch, Route } from 'react-router-dom';
 class ForumEntry extends React.Component {
```

Comment:
To be clear, a forum entry will only ever be a post, for now. We may want to refactor this, or also have a post component, that is 1:1 to the database model (author, title, body, forum, pinned<boolean>, created/updatedat)